### PR TITLE
Get multiple audited identities in a single requst.

### PR DESCRIPTION
**Pull request by defectiveAi**
## References

* #0000

### CODE REVIEW

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/GetAuditIdentityHandler.cs [hunk 1]
```diff
@@ -66,44 +67,101 @@ public async Task<GetAuditIdentityResponse> Handle(GetAuditIdentityCommand reque
...
+                        .GetAsync(user.AdministratedBy)
+                        .ConfigureAwait(false) ?? throw new InvalidOperationException($"Actor for user {user.Id} not found.");
+
+                    var organization = await _organizationRepository
+                        .GetAsync(actor.OrganizationId)
+                        .ConfigureAwait(false);
+
+                    organizationNames[user.AdministratedBy]
+                        = organizationName
+                        = organization?.Name ?? throw new InvalidOperationException($"Organization for actor {actor.Id} not found.");
```
Comment (vbelinschi) [inline source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/GetAuditIdentityHandler.cs:131]: `=` sign two times

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/GetAuditIdentityHandler.cs [hunk 2]
```diff
@@ -66,44 +67,101 @@ public async Task<GetAuditIdentityResponse> Handle(GetAuditIdentityCommand reque
...
+            NotFoundValidationException.ThrowIfNull(userIdentity, user.ExternalId.Value, $"No external identity found for user id {user.Id}.");
+
+            foundIdentities.Add(new AuditIdentityDto(user.Id.Value, $"{userIdentity.FirstName} ({userIdentity.Email})"));
+        }
 
-        return new GetAuditIdentityResponse($"{userIdentity.FirstName} ({userIdentity.Email})");
+        return foundIdentities;
     }
 
     private bool HasCurrentUserAccessToUser(User user)
```
Comment (vbelinschi) [inline source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/GetAuditIdentityHandler.cs:167]: does the method name `HasCurrentUserAccessToUser` make sense here? 😄 
looks a bit weird to me

**Review by vbelinschi — COMMENTED** [review]
one small issue and a comment/question 👍

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/GetAuditIdentityHandler.cs (hunk 1 above)
Comment (defectiveAi) [reply source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/GetAuditIdentityHandler.cs:131]: We want to assign both the dictionary and the local variable to organization?.Name, so:
`dict = localVar = source ?? exception`

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/GetAuditIdentityHandler.cs (hunk 2 above)
Comment (defectiveAi) [reply source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/GetAuditIdentityHandler.cs:167]: Better name suggestions are welcome. :D The purpose of the method is to check whether the current user can access the user they are asking to be named.

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/GetAuditIdentityHandler.cs (hunk 1 above)
Comment (vbelinschi) [reply source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/GetAuditIdentityHandler.cs:131]: aaah, read it completelythe other way

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/GetAuditIdentityHandler.cs (hunk 2 above)
Comment (vbelinschi) [reply source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/GetAuditIdentityHandler.cs:167]: maybe `HasCurrentUserAccessToActor`? 
if it doesn't make sense, just ignore it

**Review by vbelinschi — DISMISSED** [review]

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/GetAuditIdentityHandler.cs (hunk 2 above)
Comment (defectiveAi) [reply source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/GetAuditIdentityHandler.cs:167]: Hmm, maybe you just found a bug. :O Am not sure being part of actor is enough, I go ask business.

Comment (defectiveAi) [reply source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Handlers/GetAuditIdentityHandler.cs]: No bug, name updated.

**Review by vbelinschi — APPROVED** [review]

**Pull request merged**